common/Configuration.py

```diff
@@ -1,24 +1,141 @@
+import os
+import shutil
+from pathlib import Path
+
+from ._fill_in_template import fill_in_template_file
```

Comment:
From recent commits, it looks like you adopted the rule that internal functions in internal files (e.g., `_internal.py`) should have names with leading underscores (e.g., `_my_internal()`)